greenpeace/planet4-master-theme
# PLANET-7645: Implement image replacement feature

Ref: https://jira.greenpeace.org/browse/PLANET-7645

<hr>

**DESCRIPTION:** 
This PR is the first of a series in an attempt to implement a media replacement feature.
This PR replaces images from the Media Library, including the entire set of related images (thumbnail, medium, custom, etc.), while keeping the original files' name ONLY in the `uploads` folder (not in Google Storage yet).

<hr>

**TESTING (local env only):**
1. Go to the Media Library page: `/wp-admin/upload.php`
2. Select the image to replace.
3. Click on the Replace Media button.
4. Wait for the page to reload.
5. Check that the whole set of related images in the `planet4/uploads` folder was replaced.

<hr>

**NOTES:**
As this task seems complex, I plan to split it into multiple PRs and follow these steps:
1. Replace images in the `uploads` folder (done in this PR).
2. Implement a mechanism to purge the CDN service.
3. Replace the files in Google Storage (`wp-stateless`).
4. Add automated tests.
5. Implement the replacement feature for the rest of the file types (videos, documents, etc.)
